Dockerfile

```
@@ -10,6 +10,7 @@ RUN set -exo pipefail \
         --disablerepo=* \
         --enablerepo=fedora,updates \
         gcc \
+        git \
```

I think `git-core` should suffice:

```suggestion
        git-core \
```